### ISSUE TITLE
Dots of method/property invocations should be placed on same line as invoked method/property

### ISSUE THREAD

**Issue by RalfKoban** [RalfKoban/MiKo-Analyzers#522]
To ease reading, we should file an issue when dots of method/property invocations are not placed on the same line as the invoked method/property itself.

Following code should be reported:
```c#
    public IEnumerable<int> DoSomething()
    {
        return Enumerable.
                    Empty<int>().
                    OrderBy(_ => _).
                    ThenBy(_ => _).
                    ToList();
    }
```

Instead, following code should NOT be reported:
```c#
    public IEnumerable<int> DoSomething()
    {
        return Enumerable
                    .Empty<int>()
                    .OrderBy(_ => _)
                    .ThenBy(_ => _)
                    .ToList();
    }
```
